src/base/bittorrent/session.cpp

```diff
@@ -2934,6 +2941,45 @@ void Session::setAnnounceToAllTiers(const bool val)
     }
 }
 
+int Session::peerTurnover() const
+{
+    return m_peerTurnover;
+}
+
+void Session::setPeerTurnover(const int val)
+{
+    if (val != m_peerTurnover) {
```

Comment:
Please change to this style:
```c++
if (val == m_peerTurnover)
    return;

// ...
```

Reply:
Will do.

Reply:
Will do - but I copied this style from other similar functions.